tests/unit/DispatchService.unitTest.ts

```diff
@@ -132,9 +132,10 @@ describe('Dispatch Service', () => {
 
     describe('with valid REMOVE event type', () => {
       const event = {
-        eventType: 'REMOVE',
-        body: {
+        eventName: 'REMOVE',
+        dynamodb: {
           Keys: { testResultId: { S: '123' } },
+          NewImage: {}
```

Comment:
See above.  I think that for Remove events this is omitted completely.

Reply:
Addressed: https://github.com/dvsa/cvs-tsk-edh-dispatcher/pull/10/commits/6b247264e1f38437171ef46df1f25a7f4df57d33